src/main/java/pl/put/poznan/transformer/app/TextTransformerApplication.java

```diff
@@ -18,7 +26,10 @@
      *
      * @param args List of arguments to be used by spring application
      */
-    public static void main(String[] args) {
+    public static void main(String[] args) throws DomainException {
+        Text test = new InverseText (new CapitalizeText (new FromShortcut(new  NumberExpandText(new UpperText(new TextImpl("Cześć, to testowy tekst nr 654.43 i ciekawe czy wyjdzie np. ;)"))))));
+        System.out.println(test.transform());
+
```

Comment:
Co tu się stało?

Reply:
Czkawka gita, miałem to w swoim mainie, usunę to oczywiście.